examples/42_test_helix.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2024 NREL
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# See https://floris.readthedocs.io for documentation
+
+import matplotlib.pyplot as plt
+import numpy as np
+import yaml
+
+import floris.flow_visualization as flowviz
+from floris import FlorisModel
+
+
+"""
+Example to test out using helix wake mixing of upstream turbines.
+Helix wake mixing is turned on at turbine 1, off at turbines 2 to 4;
+Turbine 2 is in wake turbine 1, turbine 4 in wake of turbine 3.
+"""
+
+# Grab model of FLORIS and update to helix-enabled turbines
+fmodel = FlorisModel("inputs/emgauss_iea_15mw.yaml")
```

Comment:
Could we instead add the necessary Helix parameters to inputs/emgauss.yaml; load inputs/emgauss.yaml; and then `set` the `turbine_types` to `["iea_15mw"]`? That would save us adding another example input yaml to keep track of.

Reply:
I get where you're coming from, but for now I've left it in for the following reason: the helix parameters are tuned specifically for 15MW simulations, so it feels weird to me to define these helix expansion parameters in a file defined for a turbine for which they aren't calibrated.

Reply:
That makes sense. Perhaps, then, we could instead call this file emgauss_helix.yaml, and have it specify the IEA_15MW as it's turbine (as it already does)? We could then put in the description (or just a comment at the top of the file) that the helix parameters specified are tuned to work with the IEA 15MW turbine.